justfile

```diff
@@ -97,7 +97,8 @@ run args="":
       --dart-define="BRANCH=$(git rev-parse --abbrev-ref HEAD)" \
       --dart-define="REGTEST_FAUCET=http://localhost:8080" \
       --dart-define="REGTEST_MAKER_FAUCET=http://localhost:18000/api/pay-invoice" \
-      --dart-define="HEALTH_CHECK_INTERVAL_SECONDS=2"
+      --dart-define="HEALTH_CHECK_INTERVAL_SECONDS=2" \
+      | tee -a ../data/app.log
```

Comment:
I'm against this change. I like reading the logs in my terminal. Can't you just add this when you call `just run`?

Reply:
+1

Reply:
This should provide a superset of the previous functionality, though, as tee will redirect the output to `../data/app.log` as well as stdout, i.e you can still read it from stdout

Reply:
left is tailing the logfile, right is running `just run` itself ![image](https://github.com/get10101/10101/assets/6688948/86a37c9d-c335-465c-bb84-7479172ceb66)

Reply:
Yes, it *is* still giving us the terminal as it was. Note that this is super useful, because we often want to be in control (e.g. by doing hot restart, taking a screenshot or whatever).

So there's no downside for the terminal lovers :-)